WhlgPublicWebsite/Startup.cs

```diff
@@ -160,7 +160,7 @@ private void ConfigureCookieService(IServiceCollection services)
         {
             options.Cookie.Name = "Antiforgery";
             options.Cookie.HttpOnly = true;
-            options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
```

Comment:
this change could be of concern. there's a ticket [PC-1695](https://beisdigital.atlassian.net/browse/PC-1695) which looks into more whether it's okay to use `SameAsRequest`. without changing this the site can't work locally as setting up a local https cert through docker is difficult

Reply:
Is there any way we can use e.g. environment variables to decide which to set it as? In deployed environments we really should be using `Always`.